api/network.go

```diff
@@ -0,0 +1,11 @@
+package api
+
+// IsIngressNetwork check if the network is an ingress network
+func IsIngressNetwork(nw *Network) bool {
```

Comment:
Could you put these helper functions in an `api/network` package? We've tried to reserve `api` for autogenerated code, though there are some exceptions now.